### PR TITLE
LPS-115749 Add AccoutGroupsAccountEntryRel

### PR DESCRIPTION
**Pull request by alee8888**
https://issues.liferay.com/browse/LPS-115749

@drewbrokke @jonmak08

### CODE REVIEW

**Comment by alee8888** [comment]
ci:test:sf

File: modules/apps/account/account-service/src/main/java/com/liferay/account/service/impl/AccountGroupLocalServiceImpl.java [hunk 1]
```diff
@@ -65,6 +66,24 @@ public AccountGroup addAccountGroup(
 		return accountGroupPersistence.update(accountGroup);
 	}
 
+	@Override
+	public AccountGroup deleteAccountGroup(
+		AccountGroup accountGroup, long accountGroupId) {
```
Comment (drewbrokke) [inline modules/apps/account/account-service/src/main/java/com/liferay/account/service/impl/AccountGroupLocalServiceImpl.java:71]: I think you can just get away with only the `accountGroup` here. No need for the `accountGroupId`.

**Comment by alee8888** [comment]
ci:test:sf

**Comment by alee8888** [comment]
ci:test:sf

File: modules/apps/account/account-test/src/testIntegration/java/com/liferay/account/service/test/AccountGroupAccountEntryRelLocalServiceTest.java [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+		_accountEntry = AccountEntryTestUtil.addAccountEntry(
+			_accountEntryLocalService);
+
+		_accountGroup = AccountGroupTestUtil.addAccountGroup(
+			_accountGroupLocalService, RandomTestUtil.randomString(),
+			RandomTestUtil.randomString());
+	}
+
+	@Test
+	public void testAccountGroupAccountEntryRel() throws Exception {
```
Comment (pei-jung) [inline modules/apps/account/account-test/src/testIntegration/java/com/liferay/account/service/test/AccountGroupAccountEntryRelLocalServiceTest.java:69]: I guess this is a typo. It should be test**Add**AccountGroupAccountEntryRel

File: modules/apps/account/account-service/src/main/java/com/liferay/account/internal/model/listener/AccountEntryModelListener.java [hunk 3]
```diff
@@ -43,6 +45,20 @@
 	public void onAfterRemove(AccountEntry accountEntry)
 		throws ModelListenerException {
 
+		// Account Groups
```
Comment (pei-jung) [inline modules/apps/account/account-service/src/main/java/com/liferay/account/internal/model/listener/AccountEntryModelListener.java:48]: comment should be 

`// Accout Group Account Entry Rels`

**Comment by alee8888** [comment]
ci:test:sf

**Comment by pei-jung** [comment]
See https://github.com/pei-jung/liferay-portal/pull/665. Thanks!

**Pull request closed without merging**